opencga-server/src/main/java/org/opencb/opencga/server/rest/PanelWSServer.java

```diff
@@ -291,4 +292,24 @@ public Response updateAcl(
         }
     }
 
+    // TODO: Add panel fields to aggregation stats
+    @GET
+    @Path("/aggregationStats")
+    @ApiOperation(value = "Fetch catalog panel stats", response = FacetField.class)
+    public Response getAggregationStats(
+            @ApiParam(value = ParamConstants.STUDY_DESCRIPTION) @QueryParam(ParamConstants.STUDY_PARAM) String studyStr,
```

Comment:
Same here. We should add the fields we currently have in the /search and /distinct web services